ISSUE_TEMPLATE.md

```diff
@@ -14,26 +14,26 @@ You can, of course, also flag down an instructor if you run into anything.
 
 Replace `gburdell3` with your GT username anywhere it appears in this list.
 
-- [ ] Star this repository.
-- [ ] Create a fork of this repository.
-- [ ] Clone your fork to your computer.
-- [ ] Set your forked repository as a remote named `origin`.
-- [ ] Set our repository as a remote named `rj`.
-- [ ] Create a branch named your GT username (ex. `gburdell3`).
-- [ ] Use the terminal to create a folder named `2017` in the root of the repo.
-- [ ] Use the terminal to create an empty text file named `gburdell3.txt` in the `2017` folder.
-- [ ] Stage and commit the file (make sure you leave good commit messages!)
-- [ ] Create a new branch from `gburdell3` named `gburdell3-add-text`.
-- [ ] Add some ASCII art to your file (no longer than 30 lines, no more than 80 characters per line) on that branch and merge it into the `gburdell3` branch.
-- [ ] Make a merge conflict happen between `gburdell3` and a branch named `gburdell3/merge-conflict`.
-- [ ] Resolve the merge conflict.
-- [ ] Create two branches named `b1` and `b2`.
-- [ ] Commit a file called `stuff1.txt` in `b1`.
-- [ ] Commit a file called `stuff2.txt` in `b2`.
-- [ ] Take the changes from `b1` and apply them to `b2`.
-- [ ] In `b2`, delete both files and commit the changes.
-- [ ] Take the changes from `b2` and apply them to `b1`.
-- [ ] Take the changes from `b1` and apply them to the `gburdell3/add-text` branch (this should not cause a merge conflict).
-- [ ] Delete all local branches except for `master` and `gburdell3`.
-- [ ] Send your committed changes to the `gburdell3` branch on your fork
-- [ ] Create a pull request from your fork’s `gburdell3` branch to our master branch and ask one of the instructors in your session to approve it (if you try to merge it you won’t be able to because permissions). **In your pull request description, add a line that says "Fixes #xx" where #xx is the number of the issue you created.** This will auto-close your issue once your pull request is merged.
+1. [ ] Star this repository.
```

Comment:
I don't think github supports numbers for checkboxes, can you double check that they work? They don't work for me.

Reply:
The checkboxes still work but the numbers don't show up, so I suggest just leaving as bullets.